### ISSUE TITLE
Quadient DEV Environment ICM Server Settings Overwritten

### ISSUE THREAD

**Issue by richardcalf** [richardcalf/readable#1]
We have had an incident where a package was imported into ICM R15 originating from ICM R14

The package was configured to: [Overwite server settings on import]
![image](https://github.com/richardcalf/readable/assets/11016694/7c8307cf-2f17-4539-b8b7-5d0287a7c41c)

The net result of this is that ICM Service will not start up on our R15 instance. 
No matter what we do, it will not start.
![image](https://github.com/richardcalf/readable/assets/11016694/bc03da8f-9b94-4f76-99af-bc04bcb0428b)

The consequences of this is that it's absolutely not possible to log into ICM via designer, whatsoever.
![image](https://github.com/richardcalf/readable/assets/11016694/ce99d332-21f2-4102-b783-b7a0eecc6e6b)

To try and solve this issue, after inordinate amounts back and forth with Quadient Support, we decided to try and restore our instance from an AMI backup. This simply did not work because any good backups had already been overwriten. Too late and too bad.
![image](https://github.com/richardcalf/readable/assets/11016694/8361f1a7-7cc6-47d3-b4a0-e4c3cb74342e)

In a nutshell, Quadient Support are not able to help us. It's been back and forth for a very long time, over a week now.

Simply put, we need to in effect rollback our DEV R15 instance, by hook or by crook. 

I have scoured the log files for days on end but am sadly still unable, along with support, to determine how to actually fix this issue. I have a few gigs of downloaded logs. 

Finally support are asking for a copy of the ICM database, once again I have to admin that I do not know how to do precisely that.

**Comment by richardcalf** [comment]
This got sorted. I need to run through the process we took to get it solved. This will probably never happen.
